src/main/java/unicash/logic/parser/FindCommandParser.java

```diff
@@ -31,35 +36,58 @@ public class FindCommandParser implements Parser<FindCommand> {
      */
     public FindCommand parse(String args) throws ParseException {
         requireNonNull(args);
+
+        /* All prefixes are parsed first */
+        ArgumentMultimap argMultimapWithAllPrefixes = ArgumentTokenizer.tokenize(args, PREFIX_NAME,
+                PREFIX_DATETIME, PREFIX_AMOUNT, PREFIX_TYPE, PREFIX_CATEGORY, PREFIX_LOCATION);
+
+        /* If any of the invalid prefixes are present, an invalid command format message
+         * is displayed to the user, as opposed to the invalid prefixes themselves being
+         * parsed together with the valid prefixes as a standard field input. */
+        if (areAnyPrefixesPresent(
+                argMultimapWithAllPrefixes, PREFIX_DATETIME, PREFIX_AMOUNT, PREFIX_TYPE)) {
+
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+        }
+
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(
                 args, PREFIX_NAME, PREFIX_CATEGORY, PREFIX_LOCATION);
 
         String trimmedArgs = args.trim();
+
+        /* Throws a ParseException if the argument does not contain any non-whitespace characters
+         * or if there are any non-whitespace characters preceding the first valid prefix */
```

Comment:
The comment doesn't add much since it's understandable from the code behavior, see notes on commenting: https://nus-cs2103-ay2324s1.github.io/website/schedule/week5/topics.html#implementation-code-quality-comments-basic-do-not-repeat-the-obvious

Reply:
Ok sure, I've removed it